src/main/resources/data/minecolonies/recipes/blockhutsawmill.json

```diff
@@ -0,0 +1,29 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern":
+  [
+    "LBL",
+    "WWW",
+    "LLL"
+  ],
+  "key":
+  {
+    "B":
+    {
+      "item": "structurize:sceptergold"
+    },
+    "W":
+    {
+      "tag": "tfc:axes"
```

Comment:
This should probably be saws rather than axes

Reply:
Updated